INSTALL.md

```diff
@@ -1,149 +1,123 @@
-Installation overview
```

Comment:
Rather than having multiple installation guides targeted to the ordinary user, I think we should just remove this file and place this information in our docs. The information here would work in there as well, especially for niche Linux distribution users.